### ISSUE TITLE
[Supporting the platform - Logic Apps - Update connector]: In 320x256 viewport, contents present on the screen are not properly visible.

### ISSUE THREAD

**Issue by takyyon** [Azure/LogicAppsUX#6109]
### Describe the issue

User Experience:
Low vision users who use screen zoom functionality to view the content will be impacted if contents are not properly visible on the screen as they will not be able to identify the content on the screen and will be misguided.

Note: User credentials should NOT be included in the bug.

Pre-Requisite:
Achieve viewport 320 X 256 by adjusting the browser. (Open the developer's tool, resize, and zoom the browser to achieve viewport 320 X 256).

Repro Steps:
- Open https://ms.portal.azure.com/ and login via valid credentials.
- Navigate to the Left Nav bar and select 'More Services', search 'Logic App Custom Connector' and select any already created ''Logic App Custom Connector'
- Press 'Edit' button on the menu bar on top of the overview blade.
- Navigate all the controls on the General tab
- Input valid and invalid values on the General tab and Press Continue Button. Verify the errors on the page.
- Press Update Connector. Verify the errors that appear on the page
- Verify whether in 320x256 viewport, contents present on the screen are properly visible.

Actual Result:
After adjusting the viewport to 320x256 pixels, contents present on the screen are not properly visible. "general" combo box is overlapping with the text and "host" error message tooltip is truncated.

Expected Result:
After adjusting the viewport to 320x256 pixels, all contents present on the screen should be clearly and completely visible.

### Are you using new designer or old designer

New Designer (With action panels)

### Browser

Chrome, Edge

**Comment by hartra344** [comment]
This is owned by Power Automate and can't be fixed by us
